lib/bin/rake

```diff
@@ -0,0 +1,22 @@
+#!/Users/brandonfish/Documents/jruby/lib/bin/truffleruby
```

Comment:
The `bin` and `lib/bin` setup still confuses me a little. In `rbenv`, For 2.3.3 I see `#!/usr/bin/env ruby_executable_hooks` and `rbenv` jruby `#!/Users/brandonfish/.rbenv/versions/jruby-9.1.7.0/bin/jruby`.  I believe (not 100% sure) `rbenv` updates these via the `rbenv rehash` command: https://github.com/rbenv/rbenv#rbenv-rehash 
https://github.com/rbenv/rbenv/blob/d0779fc8fb90fb7268d0ee858086e92bcd316787/rbenv.d/exec/gem-rehash/rubygems_plugin.rb#L1

My local ruby dev build adds `#!/Users/brandonfish/.rbenv/versions/ruby-dev/bin/ruby`. I think jruby avoided any potential conflicts by using `#!/usr/bin/env jruby` in there repo but this implies `jruby` is on the path. Maybe the safest this to do for now would be `#!/usr/bin/env truffleruby`.  I'll keep investigating the bin files to see if I can learn more.

If we want to get these correct, it seems like they should be updated at install time.